### DISCUSSION TITLE
*_precision settings with Nous A1Z (TS011F/_TZ3000_2putqrmw) useless?

### DISCUSSION THREAD

**Discussion by MaxRower** [Koenkk/zigbee2mqtt#16931] [Devices]
Hello, I got a 4-pack of those plugs from Amazon recently, upgraded to the lastest firmware (192/3.0.0). 
I noticed, the precision settings seem to be ignored, power and voltage values are always without decimals, and energy with two decimals. 
Additionally the plug seems unable to measure low power, when I connect a <3W bulb, it always shows 0, even if it shows a current >0.
When connecting a raspberry pi, the power jumps between 0, 1, 2 and 3w.

Is this the expected behaviour for those plugs, or did I get faulty ones?
The tasmota version A1T of a friend can measure and display small values such as 1.5W.

I like the decimals for small power measurements to visually determine the charging state of older non-intelligent builtin NiCd and NiMh chargers, which would charge infinitely until unplugged. When you notice the charging curve flattens out, you can disconnect the device. I still have two of those devices, recently replaced the AAA NiCd cells with AAA NiMh, and they do work fine again.

**Comment by MaxRower** [comment]
Update: I returned the plugs and got a 4 pack of A1T, which work as expected, and already ordered another 4 pack. Not zigbee but with wifi/tasmota.

**Comment by wherzig** [comment]
Hi, I have the same problem.

Nous A1Z. Version 77 = 1.0.13 . See https://github.com/Koenkk/zigbee2mqtt/discussions/14208

https://www.zigbee2mqtt.io/devices/A1Z.html describes the precesion values. And the configuration page on zigbee2mqtt allows to change it. But afer this, there are no changes to the values.

Also: The desciption says: "This device supports OTA updates,.." 
But: No updates are offered by zigbee2mqtt
